pkg/controller/command/presentproof/command.go

```diff
@@ -164,7 +164,7 @@ func (c *Command) SendRequestPresentation(rw io.Writer, req io.Reader) command.E
 		return command.NewValidationError(InvalidRequestErrorCode, errors.New(errEmptyRequestPresentation))
 	}
 
-	if err := c.client.SendRequestPresentation(args.RequestPresentation, args.MyDID, args.TheirDID); err != nil {
+	if _, err := c.client.SendRequestPresentation(args.RequestPresentation, args.MyDID, args.TheirDID); err != nil {
```

Comment:
Shouldn't we support the correlation in controller as well ? Looks like this PR just addresses the SDK binding.

Reply:
The existing `Actions` stuff is more appropriate for the controller.

Reply:
Actions is an array. Not sure how to correlate the request. In the [bdd tests](https://github.com/hyperledger/aries-framework-go/blob/master/test/bdd/pkg/presentproof/presentproof_controller_steps.go#L262-L276) we are taking the first element.

Reply:
@rolsonquadras added TODO